pkgs/by-name/gh/gh-gei/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, buildDotnetModule
+}:
+
+buildDotnetModule rec {
+  pname = "gh-gei";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "github";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-F1sxT9wh/K6VP7n1SlmmvmHlcgxDJw6Rht2hPIiRFjE=";
+  };
+
+  projectFile = "src/gei/gei.csproj";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
```

Comment:
```suggestion
  nugetDeps = ./deps.nix; # File generated with `nix-build -A gh-gei.passthru.fetch-deps`.
```

Reply:
Implemented in 0cc66c26bc8d330f45e7cf11396c46705ca2d527.